C/CUDA/CUDA_Runtime/build_tarballs.jl

```diff
@@ -157,11 +157,19 @@ for version in CUDA.cuda_full_versions
         augmented_platform["cuda"] = CUDA.platform(version)
 
         should_build_platform(triplet(augmented_platform)) || continue
-        push!(builds,
-              (; dependencies=[Dependency("CUDA_Driver_jll"; compat="0.7")],
-                 script, platforms=[augmented_platform], products=get_products(platform),
-                 sources=get_sources("cuda", components; version, platform)
-        ))
+        if Base.thisminor(version) == v"10.2"
+            push!(builds,
+                (; dependencies=get_dependencies(),
```

Comment:
@maleadt Should get_dependencies include CUDA_Driver_jll for CUDA 10.2?

Reply:
I guess I was just trying to minimize the "amount of 10.2" that were getting into the clean, 11+ files...
 
And CUDA_Driver_jll is ready for 10.2, right?

Reply:
CUDA_Driver_jll is just a package that attempts to load a more recent driver, and if it does, the 10.2 runtime won't get used. Doesn't matter for the devices you're targeting, though.

Reply:
OK. I have removed the `get_dependencies` function, and added CUDA_Driver_jll as a dep.